features/minion/core/impl/src/test/java/org/opennms/features/minion/HeartBeatMinionBluePrintIT.java

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R). Copyright (C) 2015-2016 The OpenNMS Group,
+ * Inc. OpenNMS(R) is Copyright (C) 1999-2016 The OpenNMS Group, Inc.
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc. OpenNMS(R)
+ * is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version. OpenNMS(R) is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details. You should have received a copy of the GNU Affero
+ * General Public License along with OpenNMS(R). If not, see:
+ * http://www.gnu.org/licenses/ For more information contact: OpenNMS(R)
+ * Licensing <license@opennms.org> http://www.opennms.org/
+ * http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.features.minion;
+
+import java.util.Dictionary;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.camel.Component;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.component.seda.SedaComponent;
+import org.apache.camel.test.blueprint.CamelBlueprintTestSupport;
+import org.apache.camel.util.KeyValueHolder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.opennms.core.test.OpenNMSJUnit4ClassRunner;
+import org.opennms.minion.core.api.MinionIdentity;
+import org.opennms.minion.core.impl.MinionIdentityImpl;
+import org.springframework.test.context.ContextConfiguration;
+
+@RunWith(OpenNMSJUnit4ClassRunner.class)
+@ContextConfiguration(locations = {
+        "classpath:/META-INF/opennms/emptyContext.xml" })
+public class HeartBeatMinionBluePrintIT extends CamelBlueprintTestSupport {
```

Comment:
Please update this to extend _CamelBlueprintTest_ instead. See https://github.com/OpenNMS/opennms/commit/b0a59c1702dc2e502fd5b1184d5a029871c36bb7.